internal/cmd/scraper/scraper.go

```diff
@@ -40,6 +40,9 @@ type Config struct {
 	ProcessingRules                   []integration.ProcessingRule `mapstructure:"transformations"`
 	Percentiles                       []float64                    `mapstructure:"percentiles"`
 	DecorateFile                      bool
+	EmitterProxy                      string `mapstructure:"emitter_proxy"`
+	EmitterCaFile                     string `mapstructure:"emitter_ca_file"`
```

Comment:
could be useful to validate these config variables in the `validateConfig` function, e.g.: check if Proxy is set, and if so, if it's a valid URL. This way we can exit early.